services/shhext/service_test.go

```diff
@@ -106,6 +106,12 @@ func (s *ShhExtSuite) SetupTest() {
 	s.services[0].tracker.handler = newHandlerMock(1)
 }
 
+func (s *ShhExtSuite) TestInitProtocol() {
+	os.Remove("/tmp/1.v2.db")
```

Comment:
i suggest to create temporary directory for service with ioutil.TempDir in Setup and remove it here. also it is not clear from the test name why directory is removed before InitProtocol